documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -78,3 +78,39 @@ Using StrimziPodSets instead of StatefulSets provides more control over the func
 To enable the `UseStrimziPodSets` feature gate, specify `+UseStrimziPodSets` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
 
 IMPORTANT: The `UseStrimziPodSets` feature gate must be disabled when downgrading to Strimzi 0.27 and earlier versions.
+
+[id='ref-operator-use-kraft-feature-gate-{context}']
+== (Preview) UseKRaft feature gate
+
+The `UseKRaft` feature gate has a default state of _disabled_.
+
+The `UseKRaft` feature gate deploys the Kafka cluster in the KRaft (Kafka Raft metadata) mode without ZooKeeper.
+This feature gate is currently intended only for development and testing.
+
+IMPORTANT: **The KRaft mode is not ready for production in Apache Kafka or in Strimzi.**
+
+When the `UseKRaft` feature gate is enabled, the Kafka cluster is deployed without ZooKeeper.
+The `.spec.zookeeper` properties in the Kafka custom resource will be ignored, but still need to be present.
+The `UseKRaft` feature gate provides an API that configures Kafka cluster nodes and their roles.
+The API is still in development and is expected to change before the KRaft mode is production-ready.
+
+Currently, the KRaft mode in Strimzi has the following major limitations:
+
+* Moving from Kafka clusters with ZooKeeper to KRaft clusters or the other way around is not supported.
+* Upgrades and downgrades of Apache Kafka versions or the Strimzi operator are not supported.
+  Users might need to delete the cluster, upgrade the operator and deploy a new Kafka cluster.
+* The Entity Operator (including the User Operator and Topic operator) is not supported.
+  The `spec.entityOperator` property must be removed from the `Kafka` custom resource.
```

Comment:
As before highlight this in bold ... "must be removed" ... which is actually different from the ZooKeeper behaviour.